packages/mesa.rb

```diff
@@ -26,9 +26,13 @@ class Mesa < Package
   depends_on 'libxtrans'
   depends_on 'libxkbfile'
   depends_on 'wayland'
+  depends_on 'python'
```

Comment:
There is no package named `python`.  Based on your `pip install` line below, you should add `python27` as a dependency.